docs/setup-android.md

```diff
@@ -17,22 +18,47 @@
 
 In order to integrate CodePush into your Android project, please perform the following steps:
 
+### Plugin Installation for Expo (Android)
+
+React Native Code Push comes packaged with a plugin to automate some of the setup process:
+
+1. Open your Expo app config file (app.config.json or app.config.js instead of app.json).
+
+2. Insert the following into the plugins section of your config (Don't duplicate configuration, if the 'react-native-code-push' item already exists, simply add the android portion.). Create the plugins section if it doesn't already exist.
+
+    ```javascript
+    "plugins": [
+      [
+        'react-native-code-push',
+        {
+          android: {
+            CodePushDeploymentKey: 'YOUR_ANDROID_CODE_PUSH_KEY',
```

Comment:
Please add code sign support too

https://github.com/microsoft/react-native-code-push/blob/master/docs/setup-android.md#code-signing-setup

Reply:
Ahhh, we are doing that with fastlane cryptex and don't need it as part of the plugin. I can try to add that when I get a chance.

Reply:
Done. I have added that option.